charts/node-analyzer/templates/_helpers.tpl

```diff
@@ -158,3 +158,24 @@ possible, but accept overrides from the chart values.
 {{- define "nodeAnalyzer.clusterName" -}}
     {{- .Values.clusterName | default .Values.global.clusterConfig.name | default "" -}}
 {{- end -}}
+
+{{/*
+Determine collector endpoint based on provided region or .Values.nodeAnalyzer.apiEndpoint
+*/}}
+{{- define "nodeAnalyzer.apiEndpoint" -}}
+    {{- if .Values.nodeAnalyzer.apiEndpoint -}}
+        {{- .Values.nodeAnalyzer.apiEndpoint -}}
+    {{- else if (eq .Values.global.sysdig.region "us1") -}}
+        {{- "secure.sysdig.com" -}}
+    {{- else if (eq .Values.global.sysdig.region "us2") -}}
+        {{- "us2.app.sysdig.com" -}}
```

Comment:
how do these urls come with different orders of app and region!?

Reply:
good question 😄 

docs for reference: https://docs.sysdig.com/en/docs/administration/saas-regions-and-ip-ranges/